UnityProject/Assets/Scripts/Managers/PlayerList.cs

```diff
@@ -302,9 +302,15 @@ public ConnectedPlayer GetByConnection(NetworkConnection connection)
 	}
 
 	[Server]
-	public List<ConnectedPlayer> GetAllByUserID(string byUserID)
+	public List<ConnectedPlayer> GetAllByUserID(string byUserID, bool offLinePlayersTo = false)
```

Comment:
offLinePlayersTo what?

Reply:
I was putting it in the context of 
"I choose the trees and I choose the flowers to"

Reply:
aaah. Maybe ``includeOffline`` so it is clearer. I don't think you need to say ``players`` since ``GetAllByUserID`` hints it.

Reply:
ok